### ISSUE TITLE
Do you plan to add X3D implementation in this repo?

### ISSUE THREAD

**Issue by Kewenjing1020** [open-mmlab/mmaction#200]
Hello, recently released paper "x3d: expanding architectures for efficient video recognition" achieves comparable accuracy but with fewer ops than slowfast. Do you plan to add X3D implementation in this repo?

**Comment by zhaoyue-zephyrus** [comment]
Hi @Kewenjing1020 , C3D has been added in the [latest commit[(https://github.com/open-mmlab/mmaction/commit/1c6fb8fadb7963f35ddf4d4279c3882116cfd757). Check if it meets your need.

As of the X3D, @kennymckormick shall we add into our timeline?

**Comment by Kewenjing1020** [comment]
@zhaoyue-zephyrus @kennymckormick Yes, C3D is added in the repo. Could you share some light-weighted models trained on Kinetics? Actually, I'd like to do some training on Charades and AVA, it seems pretraining on Kinetics impacts a lot.

**Comment by kennymckormick** [comment]
Hi, @Kewenjing1020 , recently I'm working on my research project. Perhaps I will add X3D several weeks later. You are welcome to raise a PR if you decide to implement it on your own.

**Comment by zhaoyue-zephyrus** [comment]
Hi @Kewenjing1020  The porting of X3D is close to completion. Check it out here: https://github.com/zhaoyue-zephyrus/mmaction/tree/x3d. Will merge it soon.
